packages/sanity/src/core/releases/store/types.ts

```diff
@@ -32,6 +32,23 @@ export interface ReleaseSystemDocument {
   state: ReleaseState
   publishAt?: string
 }
+
+/**
+ * @internal
+ */
+export interface ScheduledRelease extends ReleaseDocument {
+  publishAt?: string
+  metadata: ReleaseDocument['metadata'] & {
+    intendedPublishAt: string
+    releaseType: 'scheduled'
+  }
+}
+/**
+ * @internal
+ */
+export const isScheduledRelease = (release: ReleaseDocument): release is ScheduledRelease =>
```

Comment:
Hmm, the naming here is vague to me, because a scheduled release to me has `state` of `scheduled` or `scheduling`. This is like a prescheduled release, no? I wonder if we should introduce a name like `intendedScheduledRelease`?